src/platform/graphics/texture.js

```diff
@@ -91,7 +102,15 @@ class Texture {
      * @param {string} [options.name] - The name of the texture. Defaults to null.
      * @param {number} [options.width] - The width of the texture in pixels. Defaults to 4.
      * @param {number} [options.height] - The height of the texture in pixels. Defaults to 4.
-     * @param {number} [options.depth] - The number of depth slices in a 3D texture.
+     * @param {number} [options.slices] - The number of depth slices in a 3D texture, the number of textures
```

Comment:
we need to design the API to support cubemap arrays for when we get to adding those

Reply:
I'd probably look at WebGPU API for some inspirations and even naming

https://www.w3.org/TR/webgpu/#dictdef-gputextureviewdescriptor

Reply:
and https://www.w3.org/TR/webgpu/#gputexturedescriptor

Reply:
I'll have a look at the WebGPU API you mentioned  but I'd leave the cubemap arrays support for a separate PR. I'm afraid of too many cascading changes if I change too much of the internals. See my other branch I mentioned in the description

Reply:
Yep not suggesting to add a support, just to make sure the changes to the API will support it (`slices` specifically)

Reply:
I renamed the "slices" to "layers" to make it more consistent with WebGPU naming but also WebGL (framebufferTextureLayer)